packages/app-project/src/components/ProjectHeader/components/Nav/Nav.js

```diff
@@ -28,13 +29,39 @@ const StyledAnchor = styled(Anchor)`
     &[href]:hover {
       border-bottom-color: ${props.color};
     }
-    &:not([href]) {
+    &[aria-current=page] {
```

Comment:
This is a helpful selector! As a follow-up, I'll refactor the styling of active links in app-content-pages to use this rather than an `active` prop. There's always warnings about `active` being an unexpected boolean passed to elements, so good to know that `aria-current` is available instead.